packages/quantic/force-app/main/default/lwc/quanticSearchBoxInput/quanticSearchBoxInput.js

```diff
@@ -222,7 +231,9 @@ export default class QuanticSearchBoxInput extends LightningElement {
   onKeyup(event) {
     switch (event.key) {
       case keys.ENTER:
-        this.handleEnter();
+        if (!this.targetClearButton) {
+          this.handleEnter();
+        }
```

Comment:
Instead of checking the condition here, I would add the logic to check the condition inside the `handleEnter` method, in order to keep the `onKeyup` method clear and very easy to read.

Reply:
Moved it!